data-cron-loader/scripts/reload-data-from-git.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Ce script recharge les données de movies à partir d'un dépôt git
+
+IO_ERROR=3
+SCRIPT_ERROR=5
+
+DATA_OUTPUT_DIR=/shared_data/
+DATA_TMP_DIR=/temp_data/
+
+git -C "$DATA_TMP_DIR" pull || (rm -rf "$DATA_TMP_DIR*" && git clone "$GIT_REPO" "$DATA_TMP_DIR") && \
+rm -rf "$DATA_OUTPUT_DIR*" && \
+cp -r "$DATA_TMP_DIR"* "$DATA_OUTPUT_DIR" || exit $IO_ERROR
+
+echo "MATCH (n) DETACH DELETE n" | /scripts/load-data-to-neo4j.sh \
+    && /scripts/load-data-to-neo4j.sh "$DATA_OUTPUT_DIR"/data/*cypher || exit $SCRIPT_ERROR
```

Comment:
Petite suggestion pour forcer la convention *.cypher
```suggestion
    && /scripts/load-data-to-neo4j.sh "$DATA_OUTPUT_DIR"/data/*.cypher || exit $SCRIPT_ERROR
```